es-docker

```diff
@@ -20,24 +20,20 @@ sleep 15
 
 echo "Discovering other nodes in cluster..."
 # Docker swarm's DNS resolves special hostname "tasks.<service_name" to IP addresses of all containers inside overlay network
-SWARM_SERVICE_IPs=$(dig tasks.${SERVICE_NAME} +short)
-echo "Nodes of service ${SERVICE_NAME}:"
-echo "$SWARM_SERVICE_IPs"
 
 HOSTNAME=$(hostname)
-MY_IP=$(dig ${HOSTNAME} +short)
+MY_IP=$(python -c 'import socket;print socket.gethostbyname(socket.gethostname())')
 echo "My IP: ${MY_IP}"
+export MY_IP
 
-OTHER_NODES=""
-
-for NODE_IP in $SWARM_SERVICE_IPs
-do
-    if [ "${NODE_IP}" == "${MY_IP}" ];then
-        continue;
-    fi
-    OTHER_NODES="${OTHER_NODES}${NODE_IP},"
-done
-
+OTHER_NODES=$(python -c '
+import os,socket;
+es_nodes=socket.gethostbyname_ex("tasks.elasticsearch")[2]
```

Comment:
Disagree. "tasks.elasticsearch" is based on hardcoded service name "elasticsearch". Use `os.environ["SERVICE_NAME"]` instead.